TriblerGUI/widgets/trustpage/js/radial_view/radial_inspector.js

```diff
@@ -40,14 +40,21 @@ function RadialInspector(d3element, options) {
                 nodeCount: data.nodes.length,
                 linkCount: data.links.length,
                 color: self.config.node.color.range[0]
-            };
+            },
+            user = "user",
+            exploreEncouragement = "Exchange content to see more users.";
```

Comment:
`exporeEncouragement = bindings.nodeCount !== 1 ? "Click on users to explore the rest of the network." : "Exchange content to see more users."`